src/v/coproc/gen.json

```diff
@@ -27,7 +27,7 @@
     {
       "name": "heartbeat",
       "input_type": "empty_request",
-      "output_type": "empty_response"
+      "output_type": "state_size_t"
```

Comment:
where do you define this type?

Reply:
https://github.com/graphcareful/redpanda/blob/wasm-heartbeat/src/v/coproc/types.h#L80